MultimodalQnA/docker_compose/intel/cpu/xeon/README.md

```diff
@@ -72,34 +72,41 @@ export your_no_proxy=${your_no_proxy},"External_Public_IP"
 ```
 
 ```bash
-export no_proxy=${your_no_proxy}
-export http_proxy=${your_http_proxy}
-export https_proxy=${your_http_proxy}
-export EMBEDDER_PORT=6006
-export MMEI_EMBEDDING_ENDPOINT="http://${host_ip}:$EMBEDDER_PORT/v1/encode"
-export MM_EMBEDDING_PORT_MICROSERVICE=6000
-export ASR_ENDPOINT=http://$host_ip:7066
+export host_ip=${ip_address}
```

Comment:
Is this new? It doesn't seem necessary because on line 65 of this README it's setting `host_ip` and so if someone copy/pastes this it'll be overwritten by `ip_address` which probably doesn't exist?

Reply:
this must have came from the merge, I will change it